rest/data/project_test.go

```diff
@@ -347,3 +362,42 @@ func TestGetProjectAliasResults(t *testing.T) {
 	assert.Len(t, variantTasks, 1)
 	assert.Len(t, variantTasks[0].Tasks, 2)
 }
+
+func TestGetLegacyProjectEvents(t *testing.T) {
```

Comment:
Just because I'm a little nervous about schema changes, should we write a test for the backwards compatible version? (i.e. can we read the new structure into the old structure?) If you've tested that yourself though that's also fine, I feel like I have a memory of that